prbt_hardware_support/CHANGELOG.rst

```diff
@@ -6,6 +6,7 @@ Forthcoming
 -----------
 * Adopted default configuration for launchfiles
 * Renaming of STO into RUN_PERMITTED
+* Enable starting ROS without connecting to safety controller
```

Comment:
```suggestion
* Enable starting ROS without modbus connection
```

Reply:
only a suggestion. I don't want it to imply, we encourage running the robot without safety

Reply:
Makes sense, also naming wise, thank you